pyrevolve/util/supervisor/analyzer_queue.py

```diff
@@ -11,6 +11,7 @@ class AnalyzerQueue(SimulatorQueue):
 
     def __init__(self, n_cores: int, settings, port_start=11345, simulator_cmd='gzserver'):
         super(AnalyzerQueue, self).__init__(n_cores, settings, port_start, simulator_cmd)
+        self._enable_play_pause = False
```

Comment:
Not used in analyzer queue code, so why is it added here?

Reply:
Because the simulator queue can pause the simulator when not running robots, but the analyzer cannot.
(the `AnalyzerQueue` is superclassing the `SimulatorQueue`)